arrow-select/src/coalesce.rs

```diff
@@ -166,9 +171,30 @@ impl BatchCoalescer {
             // We will for sure store at least one completed batch
             completed: VecDeque::with_capacity(1),
             buffered_rows: 0,
+            biggest_coalesce_batch_size: None,
         }
     }
 
+    /// Set the biggest coalesce batch size limit
+    ///
+    /// If set to Some(limit), batches larger than this limit will bypass
+    /// coalescing and be passed through directly. If None, all batches
+    /// will be coalesced according to the target_batch_size.
+    pub fn with_biggest_coalesce_batch_size(mut self, limit: Option<usize>) -> Self {
+        self.biggest_coalesce_batch_size = limit;
+        self
+    }
+
+    /// Get the current biggest coalesce batch size limit
+    pub fn get_biggest_coalesce_batch_size(&self) -> Option<usize> {
```

Comment:
I think most of the rest of this crate doesn't use the `get_` prefix for getters

```suggestion
    pub fn biggest_coalesce_batch_size(&self) -> Option<usize> {
```

Reply:
Thank you @alamb for good suggestion!